src/ResourceManagement/RecoveryServices/RecoveryServices.Tests/ScenarioTests/VaultUsageTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Microsoft.Azure;
+using Microsoft.Azure.Management.RecoveryServices;
+using Microsoft.Azure.Management.RecoveryServices.Models;
+using Microsoft.Azure.Test;
+using System.Net;
+using Xunit;
+
+namespace RecoveryServices.Tests
+{
+    public class VaultUsageTests : RecoveryServicesTestsBase
+    {
+        string resourceGroupName = "testsitegroup";
```

Comment:
Yes that is correct. These test are supposed to be specific. Like in delete it is expected that vault exist, in retrieve we expecting vault exist. similarily in usage we are expecting vault exist.